### PR TITLE
Set namespace label; add nework policy for minio

### PR DESCRIPTION
**Pull request by kylape**

### CODE REVIEW

File: controllers/cloud.redhat.com/providers/namespace/default.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
+package pullsecrets
```
Comment (psav) [inline controllers/cloud.redhat.com/providers/namespace/default.go:1]: package name should be `namespace`

File: controllers/cloud.redhat.com/providers/namespace/default.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	"k8s.io/apimachinery/pkg/types"
+)
+
+type namespaceProvider struct {
+	providers.Provider
+}
+
+// NewNamespaceProvider returns a new End provider run at the end of the provider set.
+func NewNamespaceProvider(p *providers.Provider) (providers.ClowderProvider, error) {
+	clowderNsB, err := ioutil.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/namespace")
```
Comment (psav) [inline controllers/cloud.redhat.com/providers/namespace/default.go:19]: Won't this error out when running with a `make run`?

File: controllers/cloud.redhat.com/providers/objectstore/minio.go [hunk 3]
```diff
@@ -191,13 +197,57 @@ func NewMinIO(p *providers.Provider) (providers.ClowderProvider, error) {
...
 		raisedErr.Requeue = true
 		return nil, raisedErr
 	}
 
-	return mp, nil
+	return mp, createNetworkPolicy(p)
+}
+
+func createNetworkPolicy(p *providers.Provider) error {
+	clowderNsB, err := ioutil.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/namespace")
```
Comment (psav) [inline controllers/cloud.redhat.com/providers/objectstore/minio.go:211]: Another call that would fail on a `local` run

**Review by psav — CHANGES_REQUESTED** [review]

**Comment by kylape** [comment]
I have merged this locally.

**Pull request closed without merging**